components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityUtil.java

```diff
@@ -1478,6 +1478,28 @@ public static int getMaximumItemPerPage() {
         return maximumItemsPerPage;
     }
 
+    /**
+     * Get the Maximum Actions per Action Type to be configured.
+     *
+     * @return maximumItemsPerPage need to display.
+     */
+    public static int getMaximumActionsPerActionType() {
+
+        int maximumActionsPerActionType = IdentityCoreConstants.DEFAULT_MAXIMUM_ACTIONS_PER_TYPE;
+        String maximumActionsPerActionTypePropertyValue =
+                IdentityUtil.getProperty(IdentityCoreConstants.MAXIMUM_ACTIONS_PER_TYPE_PROPERTY);
+        if (StringUtils.isNotBlank(maximumActionsPerActionTypePropertyValue)) {
+            try {
+                maximumActionsPerActionType = Integer.parseInt(maximumActionsPerActionTypePropertyValue);
+            } catch (NumberFormatException e) {
+                maximumActionsPerActionType = IdentityCoreConstants.DEFAULT_MAXIMUM_ITEMS_PRE_PAGE;
+                log.warn("Error occurred while parsing the 'maximumActionsPerActionType' property value " +
+                        "in identity.xml.", e);
+            }
+        }
+        return maximumActionsPerActionType;
+    }
+
```

Comment:
Can you give an example how this will be defined in the deployment.toml if someone wants to change this value

Reply:
Users can customize the value by adding following config to the toml
```
[actions]
maximum_actions_per_action_type = {value}
```